src/components/TasksList/TasksQueries.tsx

```diff
@@ -0,0 +1,63 @@
+import { gql } from '@apollo/client';
```

Comment:
This is a great idea to have put all the queries in one single file. 👍 
To go even further, it could be a good idea to create a `queries` folder and put inside the `TaskQueries.tsx` file + all other queries and mutations.